lib/workflow/stores/mongo.js

```diff
@@ -372,6 +372,7 @@ function mongoStoreFactory(waterline, Promise, Constants, Errors, assert, _, san
         return waterline.graphobjects.findOne(query, options)
         .then(function(graph) {
             sanitizer.decrypt(graph.tasks[data.taskId]);
+            sanitizer.decrypt(graph.context);
```

Comment:
I think you need rebase your code, this is @lanchongyizu 's commit, should not see it in your PR.